pkg/ssh/shell/setupNode.sh

```diff
@@ -0,0 +1,18 @@
+#!/ust/bin/env bash
+#name:TASK [update apt data and install dependencies] 
```

Comment:
I believe we don't need this annotations anymore. Same for the other scripts

Reply:
this is comment in the script. We can leave it even if don't display it to the users